base/applications/mspaint/selection.cpp

```diff
@@ -65,29 +65,29 @@ ForceRefreshSelectionContents()
 
 int CSelectionWindow::IdentifyCorner(int iXPos, int iYPos, int iWidth, int iHeight)
 {
-    if (iYPos < 3)
+    if (iYPos < GRIP_SIZE)
     {
-        if (iXPos < 3)
+        if (iXPos < GRIP_SIZE)
             return ACTION_RESIZE_TOP_LEFT;
-        if ((iXPos < iWidth / 2 + 2) && (iXPos >= iWidth / 2 - 1))
+        if ((iWidth / 2 - 1 <= iXPos) && (iXPos < iWidth / 2 + 2))
```

Comment:
The -1 and +2 in here is probably also related to GRIP_SIZE.

Reply:
`2 == GRIP_SIZE - 1`. Right?

Reply:
Probably more like the interval from GRIP_SIZE - floor(GRIP_SIZE / 2) to GRIP_SIZE + ceil(GRIP_SIZE / 2).
At least that would make the most sense.

Reply:
Done.